02_bubble/brnbke/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(arr []int) []int {
+	changed := true
+	r := make([]int, len(arr))
+	copy(r, arr)
+
+	for {
+		if !changed {
```

Comment:
If you moved this if condition to the bottom of the loop body, it would still mean the same thing. But then you could also move the `changed` variable into the scope of the `for` loop.

Reply:
have moved the check as suggested so var now just in the scope of the for loops